README.md

```diff
@@ -249,19 +236,21 @@ If you aren't using autoconfiguration from a supported plugin or otherwise need
 import org.jmailen.gradle.kotlinter.tasks.LintTask
 import org.jmailen.gradle.kotlinter.tasks.FormatTask
 
-tasks.create<LintTask>("ktLint") {
+tasks.register<LintTask>("ktLint") {
     group = "verification"
     source(files("src"))
-    reports = mapOf(
-        "plain" to file("build/lint-report.txt"),
-        "json" to file("build/lint-report.json")
+    reports.set(
+        mapOf(
+            "plain" to file("build/lint-report.txt"),
+            "json" to file("build/lint-report.json")
+        )
     )
 }
 
-tasks.create<FormatTask>("ktFormat") {
+tasks.register<FormatTask>("ktFormat") {
     group = "formatting"
     source(files("src"))
-    report = file("build/format-report.txt")
+    report.set(file("build/format-report.txt"))
```

Comment:
Report is a `RegularFileProperty` and can't be reassigned, there is no `setReport` method (like in `group =` case above)